components/HowToPanel.tsx

```diff
@@ -88,13 +98,16 @@ const HowToPanel = (props) => {
                 <input
                   id='copy-code'
                   className='input-group-element'
-                  readOnly='readOnly'
+                  readOnly={true}
                   value={`<Image src='${iconName}.svg'/>`}
                 />
                 <Button
                   type='button'
                   onClick={() => {
-                    document.getElementById('copy-code').select()
+                    const element = document.getElementById(
```

Comment:
This snippet seems to be duplicated in multiple places. Can we make this a function?